qdrant_client/async_qdrant_remote.py

```
@@ -261,6 +261,18 @@ async def close(self, grpc_grace: float | None = None, **kwargs: Any) -> None:
         self._closed = True
 
+    async def server_info(self) -> types.VersionInfo | None:
+        """Return the Qdrant server's full ``VersionInfo`` model (with
+        ``title``, ``version``, and ``commit``). Awaits the REST ``/``
+        endpoint via the existing async service API binding.
+
+        On any failure (timeout, connection refused, non-2xx, API
+        error), returns ``None`` rather than raising."""
+        try:
+            return await self.http.service_api.root()
+        except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)